payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt

```diff
@@ -632,7 +632,7 @@ class LpmRepository constructor(
             DefaultIsFinancialConnectionsAvailable(),
         // Whether to enable ACHv2 in the deferred flow.
         // To be deleted when https://jira.corp.stripe.com/browse/BANKCON-6731 is completed.
-        val enableACHV2InDeferredFlow: Boolean = BuildConfig.DEBUG,
```

Comment:
Can you point me to the code where we set this to `PaymentSheet.enableACHV2InDeferredFlow`?  Or how `PaymentSheet.enableACHV2InDeferredFlow` interacts with this in general?

Reply:
This is only used [here](https://github.com/stripe/stripe-android/blob/master/payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt#L436)

There isn't a dynamic way to enable this.